etna/pipeline/base.py

```diff
@@ -270,10 +270,22 @@ def _forecast_prediction_interval(
             raise ValueError("Pipeline is not fitted! Fit the Pipeline before calling forecast method.")
         with tslogger.disable():
             _, forecasts, _ = self.backtest(ts=self.ts, metrics=[MAE()], n_folds=n_folds)
-        forecasts = TSDataset(df=forecasts, freq=self.ts.freq)
+
+        self._forecast_borders(backtest_forecasts=forecasts, quantiles=quantiles, predictions=predictions)
+
+        return predictions
+
+    def _forecast_borders(
```

Comment:
Add test for this method

Reply:
In fact, we test this method when we run tests for `forecast(prediction interval=True, ...)`, because `backtest` tested separately.